tests/commands/test_store_api.py

```diff
@@ -282,6 +280,31 @@ def test_upload_error(client_mock, config):
     assert error2.code == "error-code-2"
 
 
+def test_upload_charmbundles_endpoint(config):
+    """Upload for charm or bundles prepares ok the endpoint and call generic _upload."""
```

Comment:
I'm a bit confused with this description, perhaps we could use something like "Test charm or bundles upload using a mock endpoint"?

Reply:
There's a redundancy when starting the tests docstrings with "Test" or "Check"... it's obvious that the test function is to test something, so I start these docstrings with what *the tested code* should do.

Improved the docstring anyway.

Reply:
Ah I see, that's a good guideline to follow. The part I had problems to parse was the "prepares ok the endpoint", but it's now clear what it's doing there. Perhaps the actual phrasing could be improved in a later change if you feel like it.